### PR TITLE
Make fields in Sort compulsory

### PR DESCRIPTION
**Pull request by ghost**
Make fields in Sort compulsory so it is less confusing for users

### CODE REVIEW

File: src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java [hunk 1]
```diff
@@ -27,15 +27,21 @@
  */
 public class SortCommandParser implements Parser<SortCommand> {
 
+    protected static final String VALUE_ASC = "ASC";
+    protected static final String VALUE_DSC = "DSC";
```
Comment (JonathanHoshi) [inline src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java:31]: Just curious, is there a need for VALUE_ASC and VALUE_DSC to be protected?

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

File: src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java (hunk 1 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java:31]: I used it in parserUtil, so can't make it private!

Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java:31]: Noted

**Review by Ardentsoul — APPROVED** [review]
LGTM!

**Pull request merged**